remotespark/sparkkernelbase.py

```diff
@@ -95,6 +132,35 @@ def _get_configuration(self):
                 self.kernel_conf_name)
             self._abort_with_fatal_error(message)
 
+    def _parse_user_command(self, code):
+        # Normalize 2 signs to 1
+        if code.startswith("%%"):
+            code = code[1:]
+
+        # When no magic, return run command
+        if not code.startswith("%"):
+            code = "%{} {}".format(self.run_command, code)
+
+        # Remove percentage sign
+        code = code[1:]
+
+        split_code = code.split(None, 1)
```

Comment:
.split(None, 1) is called several times in this method and it's not obvious what it does.  Maybe define a helper method for this?